rpcs3/Emu/RSX/GL/rsx_gl_texture.cpp

```diff
@@ -136,7 +136,7 @@ namespace
 		case CELL_GCM_TEXTURE_A1R5G5B5:
 		case CELL_GCM_TEXTURE_R5G6B5:
 		case CELL_GCM_TEXTURE_A8R8G8B8:
-		case CELL_GCM_TEXTURE_R6G5B5:
+		case CELL_GCM_TEXTURE_R6G5B5: return { GL_ALPHA, GL_GREEN, GL_RED, GL_BLUE };
```

Comment:
This one worries me. Can someone check if terraria is affected? Coz i remember tinkering with this when I first started looking at rpcs3 source and setting all those formats to AGRB will interfere. Or maybe you intended to alter R6G5B5 only?